front_end/panels/ai_chat/ui/settings/advanced/TracingSettings.ts

```
@@ -0,0 +1,225 @@
+// Copyright 2025 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import { i18nString, UIStrings } from '../i18n-strings.js';
+import { getTracingConfig, setTracingConfig, isTracingEnabled } from '../../../tracing/TracingConfig.js';
```

ESLint

[error] `../../../tracing/TracingConfig.js` import should occur before import of `../i18n-strings.js`

(import/order)